### ISSUE TITLE
test results problems

### ISSUE THREAD

**Issue by kingjames1155** [nadeemlab/CIR#15]
I started to train the Mesh Only model from scratch. The model is often not updated for several hours during training.Have you ever encountered such a problem? After training 200 epochs from the beginning, the effect will be worse than that mentioned in the paper (I guess it is because of the training equipment, for example, I only used a 3090)
 In addition,you mentioned ran multiple training with different parameters on a GPU before. What parameters have you modified

**Comment by taznux** [comment]
If you use the same data to train the model, your model performance will be saturated after 150 epochs. 
I tested some parameters in config.py, but tuning the weights of the loss function would be more effective. https://github.com/nadeemlab/CIR/blob/main/model/voxel2mesh_nodule.py#L354
